deploy-agent/deployd/client/client.py

```diff
@@ -171,10 +175,16 @@ def _read_host_info(self):
                 self._stage_type = facter_data.get(stage_type_key, None)
                 self._stage_type_fetched = True
 
+            if not self._account_id:
+                ec2_metadata = facter_data.get("ec2_metadata.identity-credentials.ec2.info", None)
+                if ec2_metadata:
+                    info = json.loads(ec2_metadata)
+                    self._account_id = info['AccountId']
```

Comment:
info.get('AccountId') just in case info might not have the key.

Reply:
sure, make sense, thanks